nsxt/resource_nsxt_vm_tags.go

```diff
@@ -30,6 +31,12 @@ func resourceNsxtVMTags() *schema.Resource {
 				Description: "Instance id",
 				Required:    true,
 			},
+			"logical_switch_id": {
+				Type:        schema.TypeString,
+				Description: "(Optional) Logical switch id",
+				Optional:    true,
+				Computed:    true,
```

Comment:
Not sure why this is marked as computed?

Reply:
Probably pasted. I've removed it in the new commit.